frontend/public/components/machine-config-pool.tsx

```diff
@@ -178,23 +219,24 @@ const MachineConfigList: React.SFC<MachineConfigListProps> = ({ obj }) => (
 
 const MachineConfigPoolDetails: React.SFC<MachineConfigPoolDetailsProps> = ({ obj }) => {
   const paused = _.get(obj, 'spec.paused');
+  const { t } = useTranslation();
   return (
     <>
       <div className="co-m-pane__body">
-        <SectionHeading text="Machine Config Pool Details " />
+        <SectionHeading text={t('machine-config-pools~Machine config pool details')} />
```

Comment:
Since Machine Config Pool is a type, I think we want to keep that capitalized. Probably we want to go with the Camel case label:

```suggestion
        <SectionHeading text={t('machine-config-pools~MachineConfigPool details')} />
```